service/src/test/resources/core-case-data-store-api/populated-case-details.json

```diff
@@ -385,19 +385,6 @@
       "internationalAuthorityInvolvement": "Yes",
       "internationalAuthorityInvolvementDetails": "French authorities were involved"
     },
-    "documents_socialWorkOther": [
```

Comment:
Why got that removed?

Reply:
Got moved to bottom of file during conflict I believe.